src/DataDictionary/reducers.js

```diff
@@ -152,7 +187,70 @@ const ddgraph = (state = ddgraphInitialState, action) => {
       tableExpandNodeID: action.nodeID,
       highlightingNode: newHighlightingNode,
       secondHighlightingNodeID: null,
-      highlightingNodeSVGElement: null,
+    };
+  }
+  case 'SEARCH_SET_IS_SEARCHING_STATUS': {
+    return {
+      ...state,
+      isSearching: action.isSearching,
+    };
+  }
+  case 'SEARCH_RESULT_UPDATED': {
+    return {
+      ...state,
+      searchResult: action.searchResult,
+      matchedNodeIDs: action.searchResultSummary.generalMatchedNodeIDs,
+      matchedNodeIDsInNameAndDescription:
+        action.searchResultSummary.matchedNodeIDsInNameAndDescription,
+      matchedNodeIDsInProperties: action.searchResultSummary.matchedNodeIDsInProperties,
+      isGraphView: true,
```

Comment:
can search only happen in the graph view? I noticed when I was in the table view and started a search it went to graph view, but it looks like that is intended?

Reply:
Yes. Currently we only designed it happens on graph view. Will have table view search function designed later.